src/components/OurTeam/OurTeamForSmallScreen.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Card } from "react-bootstrap";
+import { Linkedin, Github } from "react-bootstrap-icons";
+import { ourTeamData } from "./../../data/OurTeam/ourTeamData";
+
+const OurTeamForSmallScreen = () => {
+  return (
+    <div className="ourTeamMainDiv">
+      <h1>Our Team</h1>
+      <div className="teamMembersCardMainDiv">
+        <div className="teamMembersCard">
+          {ourTeamData.map((info, index) => {
+            return index < 2 ? (
```

Comment:
can we use filter instead of map here since u are not using all the items only the ones with index less than 2 i think

Reply:
wow great idea, i totally forgot about filter func. I'm on it